New-Bank-Group-4.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
Does the .iml file need to be in there codebase, can you please add it to the .gitignore when @rachelkberryman merges her PR in. This .iml is an extension added by intellij IDE, and shouldn't be committed to our remote repository.

Reply:
No it should not be in the repo. I messed it up somehow. I tried to create a global .gitignore file but I noticed too late that it did not work... I will make sure to add it to the .gitignore file after Rachels PR is merged.

Reply:
didn't mess up 👀  happens... usually .gitignore is located in a subfolder when opening projects with IntelliJ and paths are not defined within it as well.. so we just have to wait till the .gitignore PR is merged in